package.json

```diff
@@ -61,7 +62,8 @@
     "steal-mocha": "0.0.3",
     "steal-qunit": "^0.1.1",
     "steal-tools": "^0.16.0",
-    "testee": "^0.2.5"
+    "testee": "^0.2.5",
+    "my-steal-qunit": "git://github.com/shayaneumar/steal-qunit.git#6724346e1470e715e1871e36f784ef67de3f0782"
```

Comment:
There is already steal-qunit library which is working with qunit 1x . 
I have just created a fork of that repo and referenced it here so that both of them work together.
We should find a better way of doing this, may be the way we have done it for jasmine1 and jasmine2.